sklearn/dummy.py

```diff
@@ -403,6 +417,21 @@ class DummyRegressor(MultiOutputMixin, RegressorMixin, BaseEstimator):
 
     n_outputs_ : int,
         Number of outputs.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.datasets import make_regression
+    >>> from sklearn.dummy import DummyRegressor
+    >>> X, y = make_regression(n_features=10)
```

Comment:
Can we use an example where `y` is clearly shown, so a reader can easily calculate the mean to understand how `DummyRegressor(strategy="mean")` works?

For example, setting `y = [1.0, 2.0, 6.0, 11.0]`, where the mean is 5.0.

Reply:
Good point. Done